### PR TITLE
corrections

### PR DESCRIPTION
**Pull request by romdotdog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,74 +1,82 @@
...
+
+## Stuff about me
+
+- I code on an android phone 📱.
+- I don't like GUIs.
+- I don't write cross-platform C code unless I have or someone asks me to.
+- I use gruvbox theme.
+- I like math, physics, biology and chemistry.
+- I like reading books but I hate literature.
+- Elon Musk is cool.
```
Comment (romdotdog) [inline README.md:13]: > Elon Musk is cool.

ಠ_ಠ

File: README.md [hunk 2]
```diff
@@ -1,74 +1,82 @@
...
+I am Fabricio Pashaj, a 16 year old coding addict. I started to learn HTML by the middle of February, 2020. My favourite editor is Vim (just Vim, without the `N` at the begining).
+
+## Stuff about me
+
+- I code on an android phone 📱.
+- I don't like GUIs.
+- I don't write cross-platform C code unless I have or someone asks me to.
+- I use gruvbox theme.
+- I like math, physics, biology and chemistry.
+- I like reading books but I hate literature.
```
Comment (romdotdog) [inline README.md:12]: What does this mean?

File: README.md [hunk 3]
```diff
@@ -1,74 +1,82 @@
...
+- CSS<br />
+  I only remember some stuff, because, like HTML ^...
+- JavaScript<br />
+  Mostly NodeJS and a little in-browser. I never write it by hand anymore except for tests.
+- TypeScript<br />
+  I'd say I know it on an intermediate level.
+- AssemblyScript<br />
+  The language I feel the most comfortable with when compiling to WASM.
+- V(lang)<br />
+  Quit because of trash community.
```
Comment (romdotdog) [inline README.md:58]: Consider removing the Vlang section

File: README.md [hunk 4]
```diff
@@ -1,74 +1,82 @@
...
+- Java (android dev)<br />
+  Made an app once. It had bugs and was dropped because I didn't know how to do debugging at the time.
+- Elixir & Erlang<br />
+  I know how to use them, but I can't find a use case for them.
+- Python<br />
+  I have actively used it before for a variety of things, now I just use it for scripts.
+- Bash<br />
+  .bashrc and other small scripts.
+- WebAssembly-Text (WAT)<br />
```
Comment (romdotdog) [inline README.md:67]: Add something for WebAssembly Text?

**Review by romdotdog — COMMENTED** [review]
Overall, some stuff to consider

File: README.md (hunk 1 above)
Comment (fabricio-p) [reply README.md:13]: yes

File: README.md (hunk 2 above)
Comment (fabricio-p) [reply README.md:12]: I don't write cross-platform C code unless [I have/someone tells me] to

File: README.md (hunk 1 above)
Comment (fabricio-p) [reply README.md:13]: what's wrong with that

File: README.md (hunk 2 above)
Comment (romdotdog) [reply README.md:12]: Sorry, I meant the 
> I like reading books but I hate literature.

part

Comment (fabricio-p) [reply README.md:12]: literature == ELA for you && people may suppose that I like literature if I like reading books (imho)

Comment (romdotdog) [reply README.md:12]: I see, you may want to put `I like reading books but I hate literature class.`

**Pull request merged**